README.md

```diff
@@ -125,3 +125,7 @@ yarn
 yarn build
 yarn publish
 ```
+
+---
+
+## [Changelog](https://docs.google.com/document/d/1G7tknW4cSXJvF00TTHjzCH-4-4dvR1k6G8MQMNba5M8/edit?usp=sharing)
```

Comment:
@alfonso-paella placed at the bottom of the readme. Do you think it makes sense to place it a bit higher?

Reply:
We should eventually have a proper documentation site... for now I think this is fine because the majority of devs seeing this readme will be new